src/main/java/practice/StreamPractice.java

```diff
@@ -1,20 +1,34 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
 import model.Candidate;
 import model.Person;
 
 public class StreamPractice {
+    private final CandidateValidator candidateValidator = new CandidateValidator();
     /**
      * Given list of strings where each element contains 1+ numbers:
      * input = {"5,30,100", "0,22,7", ...}
      * return min integer value. One more thing - we're interested in even numbers.
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        try {
+            return numbers.stream()
+                    .map(s -> s.split(","))
+                    .flatMap(s -> Arrays.stream(s))
+                    .map(Integer::valueOf)
+                    .filter(n -> n % 2 == 0)
+                    .mapToInt(Integer::valueOf)
+                    .min()
+                    .getAsInt();
+        } catch (RuntimeException e) {
+            throw new RuntimeException("Can't get min value from list: method_input_list " + e);
+        }
```

Comment:
The try-catch construction is not needed here - just use .orElseThrow()
```suggestion
        return numbers.stream()
                .map(s -> s.split(","))
                .flatMap(Arrays::stream)
                .map(Integer::parseInt)
                .filter(n -> n % 2 == 0)
                .sorted()
                .findFirst()
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: "
                                                                                  + numbers));
```